This has been filed as Quagga community bug 941. This patch may be removed
when upgrading to a version of Quaggawith bug 941 fixed.

*** isisd/isis_main.c
--- isisd/isis_main.c
*************** main (int argc, char **argv, char **envp
*** 269,275 ****
    /* Command line argument treatment. */
    while (1)
      {
!       opt = getopt_long (argc, argv, "df:i:hA:p:P:u:g:vC", longopts, 0);
  
        if (opt == EOF)
  	break;
--- 269,275 ----
    /* Command line argument treatment. */
    while (1)
      {
!       opt = getopt_long (argc, argv, "df:i:hA:P:u:g:vC", longopts, 0);
  
        if (opt == EOF)
  	break;
